src/components/AboutUs/Teams.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import { Padding } from 'styled-components-spacing'
+
+import { Row, Col, Grid } from '../grid'
+import { SectionTitle } from '../Typography'
+import Tab, { Tabs } from '../Common/Tab'
+import GreyBG from '../GreyBG'
+import StaffCard from './StaffCard'
+
+const Team = ({ data }) => {
+  if (!data) {
+    return null
```

Comment:
no really sure what you mean... I think we don't want a situation where it renders nothing, but in case the provided `teams` is empty, then the `currentTeam` wound be `undefined` and I'm making an early return just to avoid running into error